

using Two_Exception_CatchBlock_understanding;

Employee20 employee20 = new Employee20();

employee20.divide_advance();

